ecosystem/python/sdk/examples/rotate-key.py

```diff
@@ -0,0 +1,105 @@
+import asyncio
+import subprocess
+import time
+
+from aptos_sdk.account import Account, RotationProofChallenge
+from aptos_sdk.account_address import AccountAddress
+from aptos_sdk.authenticator import Authenticator, MultiEd25519Authenticator
+from aptos_sdk.bcs import Serializer
+from aptos_sdk.async_client import FaucetClient, RestClient
+from aptos_sdk.transactions import (
+    EntryFunction,
+    RawTransaction,
+    Script,
+    ScriptArgument,
+    SignedTransaction,
+    TransactionArgument,
+    TransactionPayload,
+)
+from aptos_sdk.type_tag import StructTag, TypeTag
+
+from .common import FAUCET_URL, NODE_URL
+
+WIDTH = 19
+
+def truncate(address: str) -> str:
+    return address[0:6] + '...' + address[-6:]
+
+def format_account_info(account: Account) -> str:
+    vals = [str(account.address()), account.auth_key(), account.private_key.hex()]
+    return ''.join([truncate(v).ljust(WIDTH, ' ') for v in vals])
```

Comment:
Why do we want to truncate address, auth_key, and private_key instead of showing the actual values? I might prefer seeing the whole addr, but that probably just me :)

Reply:
That's just so it's easy to view in the docs, otherwise the strings are too long to all fit on one line to see the changes. Run the docs locally and go to the key rotation page and you'll see what I mean! If it's confusing that way still lmk